test/parallel/test-dotenv-edge-cases.js

```diff
@@ -200,4 +200,24 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 9);
     assert.match(child.stderr, /bad option: --env-file-ABCD/);
   });
+
+  it('should handle invalid syntax in .env file', async () => {
```

Comment:
FWIW I still believe making a fixtures file and call a subprocess is pure overhead, we should just test what we need to test

Reply:
but other use it so for this one should I write it inline ?

Reply:
if inline should I use `util.parseEnv`

Reply:
Yes I think it'd make more sense (to write it inline and to use `util.parseEnv`)

Reply:
okay I'm going to do that. Agree with you

Reply:
Done !